osbs/build/build_request.py

```diff
@@ -57,7 +56,7 @@ def __init__(self, build_json_store, inner_template=None,
         self._customize_conf = None  # site customize conf for _inner_template
         self._dj = None
         self._resource_limits = None
-        self._openshift_required_version = parse_version('1.0.6')
+        self._openshift_required_version = sanitize_version('1.0.6')
```

Comment:
I think we still want the "raw" return value of `parse_version` here. By always sanitizing this value, we lose the main benefit of `parse_version` which is the ability to compare versions. The code right now doesn't actually make use of this value, but it did in the past and will likely use it again in the future.

The only place that we want to sanitize this value is in the `render_orchestrate_build` method because the value is encoded as json. In this case, we need to get back to the raw string value. But anywhere else in the code it should be using whatever `parse_version` returns.